### PR TITLE
[CST] Remove EVSS code from FilesPage component

### PR DESCRIPTION
**Pull request by jerekshoe**
## Summary
Removing code related to EVSS from the `FilesPage` component. Deleted some of the EVSS components that are no longer used as well:
* `src/applications/claims-status/components/evss/AdditionalEvidenceItem.jsx`
* `src/applications/claims-status/components/evss/FilesPageContent.jsx`
* `src/applications/claims-status/components/evss/SubmittedTrackedItem.jsx`
* `src/applications/claims-status/tests/components/evss/AdditionalEvidenceItem.unit.spec.jsx`
* `src/applications/claims-status/tests/components/evss/SubmittedTrackedItem.unit.spec.jsx`

## Related issue(s)
- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#75680

## Testing done
Updated some of the test data; Removed tests / test files that were no longer relevant (because of EVSS removal)

### Files tab with cst_use_claim_details_v2 toggle ON
Screenshot 2024-03-12 at 11 20 09 AM

### Files tab with cst_use_claim_details_v2 toggle OFF
Screenshot 2024-03-12 at 3 07 32 PM

### Tab title
Screenshot 2024-03-12 at 11 21 05 AM

## What areas of the site does it impact?
Claim Status Tool

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by samcoforma — DISMISSED** [review]
LGTM

File: src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx [hunk 1]
```diff
@@ -124,112 +123,38 @@ describe('<FilesPage>', () => {
         expect($('.documents-filed-container', container)).to.exist;
         expect($('.claims-requested-files-container', container)).to.not.exist;
       });
-
-      it('should render files page with an EvidenceAlerts section without alerts when using evss', () => {
```
Comment (pmclaren19) [inline src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx:128]: 👍

File: src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx [hunk 2]
```diff
@@ -124,112 +123,38 @@ describe('<FilesPage>', () => {
...
-        const filesPage = $('#tabPanelFiles', container);
-
-        expect(filesPage).to.exist;
-        expect($('.claim-file-header-container', container)).to.not.exist;
-        expect(getByTestId('additional-evidence-page')).to.exist;
-        expect($('.documents-filed-container', container)).to.not.exist;
-        expect($('.claims-requested-files-container', container)).to.not.exist;
-      });
-
-      it('should render files page with an EvidenceAlerts section with alerts when using evss', () => {
```
Comment (pmclaren19) [inline src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx:160]: 👍

File: src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx [hunk 3]
```diff
@@ -270,175 +195,4 @@ describe('<FilesPage>', () => {
     );
     expect(tree.props.children).to.be.null;
   });
-
-  it('should hide requested files when closed', () => {
-    const claim = {
```
Comment (pmclaren19) [inline src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx:275]: I think we want this test we just need to update the claim for lighthouse

File: src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx [hunk 4]
```diff
@@ -270,175 +195,4 @@ describe('<FilesPage>', () => {
...
-        open: false,
-        eventsTimeline: [],
-      },
-    };
-    const tree = SkinDeep.shallowRender(<FilesPage claim={claim} />);
-
-    expect(tree.subTree('RequestedFilesInfo')).to.be.false;
-  });
-
-  it('should show requested files when open', () => {
```
Comment (pmclaren19) [inline src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx:286]: I think we want this test we just need to update the claim for lighthouse

File: src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx [hunk 5]
```diff
@@ -270,175 +195,4 @@ describe('<FilesPage>', () => {
...
-        open: true,
-        eventsTimeline: [],
-      },
-    };
-    const tree = SkinDeep.shallowRender(<FilesPage claim={claim} />);
-    const content = tree.dive(['FilesPageContent']);
-    expect(content.subTree('RequestedFilesInfo')).not.to.be.false;
-  });
-
-  it('should render ask va to decide component', () => {
```
Comment (pmclaren19) [inline src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx:298]: I think we want this test we just need to update the claim for lighthouse

File: src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx [hunk 6]
```diff
@@ -270,175 +195,4 @@ describe('<FilesPage>', () => {
...
-    };
-
-    const tree = SkinDeep.shallowRender(
-      <FilesPage params={{ id: 2 }} claim={claim} />,
-    );
-    const content = tree.dive(['FilesPageContent']);
-    expect(content.everySubTree('AskVAToDecide')).not.to.be.empty;
-  });
-
-  it('should display turned in docs', () => {
```
Comment (pmclaren19) [inline src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx:322]: I think we want this test we just need to update the claim for lighthouse

File: src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx [hunk 7]
```diff
@@ -270,175 +195,4 @@ describe('<FilesPage>', () => {
...
-        ],
-      },
-    };
-
-    const tree = SkinDeep.shallowRender(<FilesPage claim={claim} />);
-    const content = tree.dive(['FilesPageContent']);
-    expect(content.everySubTree('SubmittedTrackedItem').length).to.equal(1);
-  });
-
-  it('should display additional evidence docs', () => {
```
Comment (pmclaren19) [inline src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx:345]: I think we want this test we just need to update the claim for lighthouse

File: src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx [hunk 8]
```diff
@@ -270,175 +195,4 @@ describe('<FilesPage>', () => {
...
-        ],
-      },
-    };
-
-    const tree = SkinDeep.shallowRender(<FilesPage claim={claim} />);
-    const content = tree.dive(['FilesPageContent']);
-    expect(content.everySubTree('AdditionalEvidenceItem').length).to.equal(1);
-  });
-
-  it('should show never received docs as tracked items', () => {
```
Comment (pmclaren19) [inline src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx:363]: I think we want this test we just need to update the claim for lighthouse

File: src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx [hunk 9]
```diff
@@ -270,175 +195,4 @@ describe('<FilesPage>', () => {
...
-      },
-    };
-
-    const tree = SkinDeep.shallowRender(<FilesPage claim={claim} />);
-    const content = tree.dive(['FilesPageContent']);
-    expect(content.everySubTree('SubmittedTrackedItem').length).to.equal(2);
-    expect(content.everySubTree('AdditionalEvidenceItem')).to.be.empty;
-  });
-
-  it('should clear alert', () => {
```
Comment (pmclaren19) [inline src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx:397]: I think we want this test we just need to update the claim for lighthouse

File: src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx [hunk 10]
```diff
@@ -270,175 +195,4 @@ describe('<FilesPage>', () => {
...
-        message={message}
-        claim={claim}
-      />,
-    );
-    expect(clearNotification.called).to.be.false;
-    tree.subTree('ClaimDetailLayout').props.clearNotification();
-    expect(clearNotification.called).to.be.true;
-  });
-
-  it('should clear notification when leaving', () => {
```
Comment (pmclaren19) [inline src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx:421]: I think we want this test we just need to update the claim for lighthouse

File: src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx (hunk 3 above)
Comment (jerekshoe) [reply src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx:275]: Done

File: src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx (hunk 4 above)
Comment (jerekshoe) [reply src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx:286]: Done

File: src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx (hunk 5 above)
Comment (jerekshoe) [reply src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx:298]: Done

File: src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx (hunk 6 above)
Comment (jerekshoe) [reply src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx:322]: Done

File: src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx (hunk 7 above)
Comment (jerekshoe) [reply src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx:345]: Done

File: src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx (hunk 8 above)
Comment (jerekshoe) [reply src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx:363]: Done

File: src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx (hunk 9 above)
Comment (jerekshoe) [reply src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx:397]: Done

File: src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx (hunk 10 above)
Comment (jerekshoe) [reply src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx:421]: Done

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Review by pmclaren19 — APPROVED** [review]
Thanks for adding back those tests! Looks great!

**Pull request merged**